client/signup/config/flows-pure.js

```diff
@@ -357,7 +356,7 @@ export function generateFlows( {
 	if ( isEnabled( 'gutenboarding' ) ) {
 		flows.frankenflow = {
 			steps: [ 'user', 'domains', 'plans' ],
-			destination: getEditorDestination,
+			destination: getSignupDestination,
```

Comment:
Signup destination in `/checklist/SITE_SLUG`, which redirects to customer home at `/home/SITE_SLUG` for eligible sites.